src/vs/workbench/contrib/positronAssistant/browser/components/languageModelConfigComponent.tsx

```diff
@@ -19,18 +19,30 @@ interface LanguageModelConfigComponentProps {
 	onSignIn: () => void,
 }
 
+const TOS_TEMPLATE = `{0} is considered "Third Party Materials" as defined in the [Posit EULA](https://posit.co/about/eula/)
+and subject to the {0} terms of service at {1} and privacy policy at {2}.\n\n
+Your use of {0} is optional and at your sole risk.`
```

Comment:
thoughts on removing the repeated text?

```suggestion
const TOS_TEMPLATE = `{0} is considered "Third Party Materials" as defined in the [Posit EULA](https://posit.co/about/eula/)
and subject to the {0} {1} and {2}.\n\n
Your use of {0} is optional and at your sole risk.`
```

Reply:
This is the template provided from legal so I think we should keep it as is.